atlas/packages/components/src/components/Button/Button.style.ts

```diff
@@ -1,10 +1,10 @@
 import { typography, colors } from "../../theme"
 import { makeStyles, StyleFn } from "../../utils"
-import { disabled, dimensionsFromProps, log } from "../../theme/fragments"
+import { disabled, dimensionsFromProps } from "../../theme/fragments"
 
 export type ButtonStyleProps = {
 	text?: string
-	type?: "primary" | "secondary"
+	type?: "primary" | "secondary" | "tertiary"
```

Comment:
Is `tertiary` a term used when describing styles? I haven't encountered it before. `primary` and `secondary` are pretty standard, maybe here we could use a name describing the button better? Like `transparent`, `info`, etc?

Reply:
It's not super common, but you can find it paired with `primary` and  `secondary`, like here https://github.com/Semantic-Org/Semantic-UI/issues/6229